src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdServerConfiguration.cs

```diff
@@ -118,6 +119,7 @@ public void Configure(string name, OpenIddictServerOptions options)
             {
                 options.AccessTokenHandler = new JwtSecurityTokenHandler();
             }
+            options.UseReferenceTokens = IsUseReferenceTokens(settings.AccessTokenFormat, settings.UseReferenceTokens);
```

Comment:
Let's simplify things a bit:

```suggestion
            options.UseReferenceTokens = settings.UseReferenceTokens;
```

(you can also put it after `options.UseRollingTokens = settings.UseRollingTokens`)